src/js/hca/components/form-elements/ErrorableSelect.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
```

Comment:
Yes!  I reason I want to get this in so bad is because we need to enable all the current development to refactor up common components.